docs/css-modules.md

```diff
@@ -0,0 +1,22 @@
+# CSS Modules
+
+[CSS Modules](https://github.com/w3c/webcomponents/blob/gh-pages/proposals/css-modules-v1-explainer.md) are supported in both
+s.js and system.js.
+
+Note that the term CSS Modules refers to two separate things: (1) the browser spec, or (2) the webpack / postcss plugin.
+The CSS modules implemented by SystemJS are the browser spec.
+
+## Example
+```css
+/* file.css */
+.brown {
+  color: brown;
+}
+```
+
+```js
+System.import('file.css').then(function (module) {
+  const styleSheet = module.default; // A CSSStyleSheet object
+  document.adoptedStyleSheets = [...document.adoptedStyleSheets, styleSheet]; // now your css is available to be used.
+});
+```
```

Comment:
How about a section on the constructed stylesheets status?

Something like - 

```
## Constructed Style Sheets Polyfill

CSS modules return a [Constructable Stylesheet](https://developers.google.com/web/updates/2019/02/constructable-stylesheets) instance as the default export when imported.

Currently these are only available in Chrome 73+, so usage in any other browsers will require a polyfill, such as the one at https://www.npmjs.com/package/construct-style-sheets-polyfill.

_Note that this polyfill does not currently work in IE11._

The polyfill can be conditionally loaded with an approach like:

<pre>
<script src="system.js"></script>
<script>
  try { new CSSStyleSheet() }
  catch (e) {
    document.head.appendChild(Object.assign(document.createElement('script'), {
      src: 'https://unpkg.com/browse/construct-style-sheets-polyfill@2.1.0/adoptedStyleSheets.min.js'
    }));
  }
</script>
</pre>

If the polyfill is not included, CSS modules will not work in other browsers.
```

Reply:
👍 done